### ISSUE TITLE
Should I import the AuditComponents solution (governance) in addition to Core Components solution when updating the CoE?

### ISSUE THREAD

**Issue by eileenoceilleachair** [microsoft/coe-starter-kit#7700]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hi,
I'm new to CoE.
I set up CoE and installed Core Components solution and the Audit Components (Governance) solutions in August 2023 following your videos and documentation on Microsoft Learn.

Then in Jan 2024 I upgraded the CoE using the Jan 2024 release - I imported the Core Components solution and I also imported the AuditComponents solution (governance) - see below results of the import.


Solution "Center of Excellence - Core Components" imported successfully.

Solution "Center of Excellence - Governance Components" imported successfully.

I have two queries - and apologies I'm not able to figure this answer out from the notes on how to update the CoE-

First I don't make any changes- I use the solutions as they are out pf the box and don't customise anything.

Do I need to import the Audit Components (Governance) solution each time I update the CoE?

Do I need to import the Power BI CoE and Governance pbit templates every time into Power BI Desktop which I then publish to Power Bi service - Does the power bi pbit templates change?

### What solution are you experiencing the issue with?

Governance

### What solution version are you using?

Jan 2024 4.22 Core Components and 3.14 Governance solution

### What app or flow are you having the issue with?

None - this is a query regarding updating the CoE

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by RahulBojji** [comment]
@eileenoceilleachair 

We recommend you to update the kit every three months as there will be some major changes being released. Please check [this ](https://learn.microsoft.com/en-us/power-platform/guidance/coe/after-setup)for more information.

<img width="492" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/149372606/d40f9436-0198-4885-92ba-9cb7cb6f0a83">

Also, please have a look at the [release notes](https://github.com/microsoft/coe-starter-kit/releases/tag/CoEStarterKit-February2024) in every release to check in case if any issues faced by you are fixed

<img width="872" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/149372606/68ff5793-356e-4956-9d50-a940024b92f8">

This is applicable for both the audit components solution and power bi

**Comment by eileenoceilleachair** [comment]
Hi,
Thanks for the reply.
So I take it then that I should update the Audit Components (Governance) solution too every three months - so basically I should update any of the solutions I am using including the power BI pbit templates - is this correct?

**Comment by RahulBojji** [comment]
yes correct. If you are looking for any bugs fixes related to either core or governance solutions, you can look at release notes at every release and decide whether you want to upgrade the solutions or not

**Comment by Jenefer-Monroe** [comment]
Please note also that we dont produce the Audit Components (Governance) or Nurture components if there are no changes there for the month, which does happen periodically. 
So you can compare the version number with what you have locally so see if there is an upgrade.
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/27ca1d92-369c-4088-9610-cc8c1ecd3441)
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/09643279-daa0-4465-90f4-4b2f00f5cff4)
